File: 0832-flipping-an-image/0832-flipping-an-image.js
/**
 * @param {number[][]} image
 * @return {number[][]}
 */
var flipAndInvertImage = function(image) {
  const reversed_image = image.map(row => row.reverse());
  return reversed_image.map(sub => {
    return sub.map(num => num === 0 ? 1 : 0)
  })
}

File: 0832-flipping-an-image/0832-flipping-an-image.cs
public class Solution {
    public int[][] FlipAndInvertImage(int[][] image) {
        for (int i = 0; i < image.Length; i++) {
            Array.Reverse(image[i]);
            for (int j = 0; j < image[i].Length; j++) {
                image[i][j] = 1 - image[i][j];
            }
        }
        return image;
    }
}
